stock_outgoing_shipment_report/views/stock_outgoing_shipment_report_views.xml

```diff
@@ -9,6 +9,9 @@
                 <group expand="0" string="Group By">
                     <filter name="group_by_carrier" string="Carrier" context="{'group_by':'carrier_id'}"/>
                 </group>
+                <group expand="0" string="Serch By">
+                    <filter name="createdtoday" string="Created Today" domain="[('create_date','&gt;=', datetime.datetime.combine(context_today(), datetime.time(0,0,0))), ('create_date','&lt;=', datetime.datetime.combine(context_today(), datetime.time(23,59,59)))]"/>
+                </group>
```

Comment:
Please remove the group tag and move the filter to just below `<field name="carrier_id" />` (following suit to how filters are defined in standard Odoo).